credoai/lens.py

```diff
@@ -428,49 +441,40 @@ def _validate_data(self):
         col_names = list(self.data.columns)
         if self.sensitive_feature_key not in col_names:
             raise ValidationError(
-                "The provided sensitive_feature_key " + self.sensitive_feature_key + " does not exist in the provided data"
+                "The provided sensitive_feature_key " + self.sensitive_feature_key +
+                " does not exist in the provided data"
             )
         if self.label_key not in col_names:
             raise ValidationError(
-                "The provided label_key " + self.label_key + " does not exist in the provided data"
+                "The provided label_key " + self.label_key +
+                " does not exist in the provided data"
             )
 
-    @staticmethod
-    def _concat_features_label_to_dataframe(X, y, sensitive_features):
-        """A utility method that concatenates all features and labels into a single dataframe
-
-        Returns
-        -------
-        pandas.dataframe, str, str
-            Full dataset dataframe, sensitive feature name, label name
-        """
-        if isinstance(sensitive_features, pd.Series):
-            df = pd.concat([X, sensitive_features], axis=1)
-            sensitive_feature_name = sensitive_features.name
-        else:
-            df = X.copy()
-            df['sensitive_feature'] = sensitive_features
-            sensitive_feature_name = 'sensitive_feature'
+    def dev_mode(self, frac=0.1):
```

Comment:
@amrasekh , a bunch of formatting changes you can ignore. This is the function I added